src/StdChains.sol

```diff
@@ -160,9 +160,17 @@ abstract contract StdChains {
                     chain.rpcUrl = vm.envString(envName);
                 }
                 // distinguish 'not found' from 'cannot read'
-                bytes memory notFoundError =
+                bytes memory oldNotFoundError =
                     abi.encodeWithSignature("CheatCodeError", string(abi.encodePacked("invalid rpc url ", chainAlias)));
-                if (keccak256(notFoundError) != keccak256(err) || bytes(chain.rpcUrl).length == 0) {
+                // TODO: use `abi.encodeWithSelector(Vm.CheatcodeError)` on next Vm update
```

Comment:
Can we resolve this TODO on this PR or is something blocking it?

Reply:
would need to add the error type to Vm

Reply:
Is something upstream blocking that addition or can we make that change in this PR? One potential issue is I'm not sure offhand is solidity 0.6.2 supports that syntax

Reply:
Yeah that's also a problem, custom errors are in 0.8.4. I guess this is fine for now

Reply:
Yea I think this is ok, thanks for updating